.github/workflows/deploy.yml

```diff
@@ -149,7 +149,7 @@ jobs:
         path: build/html
 
   deploy:
-    if: github.repository_owner == 'ros-planning' && github.ref == 'refs/heads/main'
+    if: github.repository_owner == 'ros-planning'
```

Comment:
Oh is this going to murderize main?

Reply:
Definitely. I'm just speeding up my testing and will revert this line if this ends up working.